client/python/gradio_client/utils.py

```diff
@@ -898,13 +914,18 @@ def get_type(schema: dict):
         raise APIInfoParseError(f"Cannot parse type for {schema}")
 
 
-OLD_FILE_DATA = "Dict(path: str, url: str | None, size: int | None, orig_name: str | None, mime_type: str | None)"
-FILE_DATA = "Dict(path: str, url: str | None, size: int | None, orig_name: str | None, mime_type: str | None, is_stream: bool)"
+FILE_DATA_FORMATS = [
```

Comment:
We can mostly remove this in 4.0 when we don't need to use the old data formats to determine if a component has any filedata in its payload